tool/grind.dart

```diff
@@ -44,3 +44,21 @@ build() {
 deploy() {
   run('surge', arguments: ['-d', 'which-film.info', '-p', 'build/web']);
 }
+
+@Task('Run in browser.')
+@Depends(build)
+browser() {
+  try {
+    run('hash', arguments: ['dartium']);
```

Comment:
What does this line do? I ask because I don't have Dartium installed through homebrew so if we do have a Dartium rule I would like to have a matching one that doesn't require Dartium.